api/lib/validation/schemaValidator.js

```diff
@@ -0,0 +1,183 @@
+'use-strict'
```

Comment:
We forgot the initial_instance_count which is optional and allows you to select something in the range between min and max when a schedule triggers.